src/components/Preferences/SharedLabels.js

```diff
@@ -0,0 +1,293 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import {
+  Button,
+  Checkbox,
+  IdentityBadge,
+  Toast,
+  breakpoint,
+  font,
+  theme,
+} from '@aragon/ui'
+import {
+  IdentityContext,
+  identityEventTypes,
+} from '../IdentityManager/IdentityManager'
+import LoadingRing from '../LoadingRing'
+import { AragonType } from '../../prop-types'
+import { GU } from '../../utils'
+import { useSelected } from '../../hooks'
+
+const SharedLabels = React.memo(function SharedLabels({
+  onSave,
+  onClose,
+  labels,
+  toast,
+  wrapper,
+}) {
+  const { identityEvents$ } = React.useContext(IdentityContext)
```

Comment:
What's the difference with `Fragment`? I don't mind any way but if we weren't destructuring others why choose to do so randomly?

Reply:
That's a good point, we generally don't destruct `memo`, `Fragment`, `Component`, and `PureComponent`. Leaving the `React.` prefix helps me understand them at a glance, as they're "top-level" constructs that set the stage for a component.

With hooks, because you can only use them within a react component, I think it's clear to not have the `React.` prefix, and there also seems to be some alignment across the community on having them destructured (you see it destructured in the docs and etc).